package.json

```diff
@@ -2,22 +2,27 @@
   "name": "cozy-notes",
   "version": "1.17.0",
   "scripts": {
-    "tx": "tx pull --all || true",
-    "lint": "yarn lint:js && yarn lint:styles",
-    "lint:js": "cs lint '{src,test}/**/*.{js,jsx}'",
+    "build:browser:dev": "cs build --browser --development --ts",
+    "build:browser": "cs build --browser --ts",
+    "build:mobile": "cs build --mobile --ts",
+    "build": "yarn build:browser",
+    "cozyPublish": "cs publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})",
+    "deploy": "git-directory-deploy --directory build/ --branch ${DEPLOY_BRANCH:-build} --repo ${DEPLOY_REPOSITORY:-https://cozy-bot:$GITHUB_TOKEN@github.com/cozy/cozy-notes}",
+    "install:clean:start": "yarn install:clean && yarn start",
+    "install:clean": "rm -rf node_modules && rm -f yarn.lock && yarn install",
```

Comment:
Why are you deleting the lock file? We should not do that.

Reply:
Yes this is a mistake, updated